Model/Payment/Api/Client.php

```diff
@@ -119,7 +119,8 @@ public function authorize($breadTransactionId, $amount, $merchantOrderId = null)
                 ]
             );
             throw new \Magento\Framework\Exception\LocalizedException(
-                __('Bread authorized amount ' . $breadAmount . ' but transaction expected ' . $amount)
```

Comment:
@dfroneberger does this mean that the amounts will no longer be written to the logs? That information has been helpful when diagnosing the cause of a price mis-match.

Reply:
No we log the amounts right above this line here and that's staying the same, this is just the message that's sent back to the user, so they won't see the amounts anymore but that's it